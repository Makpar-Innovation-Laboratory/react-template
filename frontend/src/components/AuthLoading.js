import React from "react";

const AuthLoading = () => {

  return (
    <div id="auth-loading-container">
      <div id="message">
        Please wait, your user account is currently being authenticated.
      </div>
      <div id="loader-container">
        <div className="loader" id="loader-left" />
        &nbsp;
        <div className="loader" id="loader-right" />
        &nbsp;
      </div>
    </div>
  );
};

export default AuthLoading;
