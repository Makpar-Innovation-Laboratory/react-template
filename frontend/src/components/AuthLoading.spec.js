import React from "react";
import { shallow } from "enzyme";
import AuthLoading from "./AuthLoading";

describe("JSX Tests :: <AuthLoading /> Rendering", () => {

  test("Sanity Check Snapshot Rendering", () => {

    const wrapper = shallow(<AuthLoading />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
