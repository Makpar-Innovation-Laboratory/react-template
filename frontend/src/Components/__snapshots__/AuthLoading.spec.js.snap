// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSX Tests :: <AuthLoading /> Rendering Sanity Check Snapshot Rendering 1`] = `
"<div id=\\"auth-loading-container\\">
  <div id=\\"message\\">
    Please wait, your user account is currently being authenticated.
  </div>
  <div id=\\"loader-container\\">
    <div className=\\"loader\\" id=\\"loader-left\\" />
     
    <div className=\\"loader\\" id=\\"loader-right\\" />
     
  </div>
</div>"
`;
